package.json

```diff
@@ -107,8 +101,18 @@
                     "type": "boolean",
                     "default": true,
                     "description": "Enable/disable Unibeautify formatter"
+                },
```

Comment:
Please rebase with the current master branch and apply these changes on top of it